admin/taxonomy/class-taxonomy-metabox.php

```diff
@@ -119,6 +125,40 @@ private function get_content_sections() {
 		return $content_sections;
 	}
 
+	/**
+	 * Returns the metabox content for React to hook into.
+	 *
+	 * @return WPSEO_Metabox_Section
+	 */
+	private function get_content_meta_section_react() {
```

Comment:
same as above, it returns content specific to the "Content optimization" section; I'd consider some renaming.